jiraApi/jiraApi.go

```diff
@@ -192,21 +197,37 @@ func GetIssues(issueKeys []string) []models.Issue {
 }
 
 // Worklog method add worklog to issue
-func AddWorklog(key string, min uint64, com string) error {
+func AddWorklog(key string, min uint64, com string, date string, time string) (models.WorklogResp, models.WorklogAdd, error) {
+	if date == "" {
+		date = clock.Now().Format("2006-01-02")
+	}
+	if time == "" {
+		time = clock.Now().Format("15:04")
+	}
+	_, offset := clock.Now().Local().Zone()
+	sign := ""
+	if offset > 0 {
+		sign = "+"
+	}
+	udata := date + "T" + time + ":00.000" + sign + strconv.Itoa(offset)
```

Comment:
There is no validation of date and time. How about something like this:
t, err := clock.ParseInLocal("2006-01-0215:04", date+time, clock.Local)
// handle, error
udata := t.Utc().Format(clock.RFCsomething)